xml/bk_openstack_admin.xml

```diff
@@ -13206,10 +13206,7 @@ partitions have four replicas and 80 percent have three replicas.</para>
         <para>The replica count is adjustable. For example:</para>
         <screen>$ swift-ring-builder account.builder set_replicas 4
 $ swift-ring-builder account.builder rebalance</screen>
-        <para>You must rebalance the replica ring in globally distributed clusters.
-Operators of these clusters generally want an equal number of replicas
-and regions. Therefore, when an operator adds or removes a region, the
-operator adds or removes a replica. Removing unneeded replicas saves on
+        <para>Removing unneeded replicas saves on
 the cost of disks.</para>
```

Comment:
removing replicas do save on cost of disks but you loose durability.

Reply:
Please suggest alternative content for removing/not removing replicas.